docs/apps_or_lib.md

```diff
@@ -0,0 +1,66 @@
+# django-ansible-base Features
+django-ansible-base provides features as combination of django applications and python libraries.
+
+All top level folders in ansible_base are django applications for features except for the lib folder.
+
+## Determining if a new feature should be an app or a lib
+
+To determine if a feature should be a library or a django-application please consider the following criteria.
+
+If any of the following are true, you should create your new feature as a django app:
+1) Does the feature require models
+2) Does the feature require urls
+3) Does the feature provide management commands
+4) Does the feature require settings
+
+If all of these are false, your feature should be a library under the `libs` folder.
+
+## Initializing a new application
+
+### Selecting a name
+
+Please try to choose a concise descriptive name that is not an overloaded term.
+
+For example, we chose `rest_filters` instead of using a generic term like `filters`.
+
+### Initializing an application
+
+To have django initialize your application run the following commands:
+
+```
+cd ansible_base
+python ../manage.py startapp <app name>
+```
+
+This will automatically create a folder and files for you with the name of your application in ansible_base.
+
+You can leave the `__init__.py` and the `migrations` folder alone.
+
+The `admin.py` file should be deleted.
```

Comment:
If you have models you could register them in that file. If you have no use for it, then sure delete it.

Reply:
Thanks for the reminder, I wanted to update this.